keyboards/handwired/prime_exl_plus/keymaps/default/keymap.c

```diff
@@ -0,0 +1,93 @@
+    ),
+
+	LAYOUT(
+		KC_NLCK, 	          KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 				KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS,    KC_TRNS, 	  KC_TRNS, 	  KC_TRNS, 		KC_TRNS, 	  KC_TRNS, 	  KC_DEL,
+		RGB_TOG, 	          RGB_MOD, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS,				KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS,		KC_TRNS,  	KC_TRNS, 	  KC_UP, 	    KC_TRNS, 	  KC_SCLN, 	  KC_BSLS,    KC_TRNS,
+		RGB_VAI,	          RGB_VAD,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,				KC_CAPS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS,		KC_TRNS,  	KC_LEFT, 	  KC_DOWN, 		KC_RGHT,  	KC_TRNS, 	              KC_TRNS,
+		RGB_MODE_RAINBOW,	  KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,				KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 	  KC_TRNS,    KC_TRNS,		KC_TRNS, 		KC_TRNS, 		KC_TRNS,  	KC_TRNS, 	  KC_TRNS,    KC_TRNS,    KC_TRNS,
+		KC_TRNS,            KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,        KC_TRNS,    KC_TRNS,                            KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,                                        KC_TRNS,    KC_TRNS
+    ),
+
+	LAYOUT(
+		KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 				KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS,				 		KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS,
+		KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS,				KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS,			 	 		KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS,  KC_TRNS,
+		KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,				KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS,				 		KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS,	KC_TRNS,           	KC_TRNS,
+		KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,				KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 	  KC_TRNS,		KC_TRNS,						KC_TRNS,  KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS,  KC_TRNS,  KC_TRNS,
+		KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,        KC_TRNS,    KC_TRNS,                            KC_TRNS,    KC_TRNS,            KC_TRNS,  KC_TRNS,                                KC_TRNS,  KC_TRNS
+    )
+};
+
+void matrix_init_user(void) {
+  // set CapsLock LED to output and low
+  setPinOutput(B0);
+  writePinHigh(B0);
+  // set NumLock LED to output and low
+  setPinOutput(B1);
+  writePinHigh(B1);
+  // set ScrollLock LED to output and low
+  setPinOutput(B2);
+  writePinHigh(B2);
+}
+
+void led_set_user(uint8_t usb_led) {
+  if (IS_LED_ON(usb_led, USB_LED_NUM_LOCK)) {
+    writePinHigh(B1);
+  } else {
+    writePinLow(B1);
+  }
+  if (IS_LED_ON(usb_led, USB_LED_CAPS_LOCK)) {
+    writePinHigh(B0);
+  } else {
+    writePinLow(B0);
+  }
+  /*
+  if (IS_LED_ON(usb_led, USB_LED_SCROLL_LOCK)) {
+    writePinHigh(B2;
+  } else {
+    writePinLow(B2);
+  }
+  */
+}
```

Comment:
All this logic should be set up in `prime_exl_plus.c` instead of here, that way every keymap can use it out of the box.

Also, your comments indicate the LED pins should be low by default, but you're driving them high. Ideally they should be off at boot, then turned on if the corresponding Lock key is enabled (or layer, if you're using them for layer indicators).